R/JHU.R

```diff
@@ -1,5 +1,9 @@
-#' R6 Class containing specific attributes and methods for John Hopkins
-#' University data
+#' R6 Class containing specific attributes and methods for COVID-19 data
+#' provided by John Hopkins University.and used for the 2019 Novel Coronavirus
+#' Visual Dashboard opperated by the Johns Hopkins University
+#' Center for Systems Science and Engineering (JHU CSSE). Also,
+#' Supported by ESRI Living Atlas Team and the Johns Hopkins University
+#' Applied Physics Lab (JHU APL)
```

Comment:
Do they have a citation they want people to use? Might be good to say what it is for people.